.azure/gpu-benchmark.yml

```diff
@@ -24,7 +24,7 @@ pr:
       - "tests/tests_pytorch/benchmarks/**"
       - "requirements/pytorch/**"
     exclude:
-      - "requirements/pytorch/docs.txt"
```

Comment:
Same comment as for the Lite job

Reply:
how about deep structure? https://github.com/Lightning-AI/lightning/pull/15770#discussion_r1029681627